src/yadism/esf/exs.py

```diff
@@ -5,6 +5,7 @@
 from .esf import ESFInfo
 from .result import ESFResult, EXSResult
 
+INV_GEV_TO_PB = 3.8937966e8 # pb
```

Comment:
@Tanjona, the conversion between `pb` and `cm2` is almost trivial, and in any case I would store in a separate constant.

As you can see, we have now two constants, with a mantissa that is the same up to $10^{-6}$, and the difference is just the exponent (so you multiply this by `1e2` and you obtain essentially next line...)

Reply:
This does make sense! I have removed the explicit definition of the constant but rather kept the variable name in the code just to make it explicit which conversion is being done XD.